src/Components/Avatar/Avatar.js

```diff
@@ -0,0 +1,72 @@
+
+import React from 'react';
+import 'antd/dist/antd.css';
```

Comment:
This component has its own css. If so, how to proceed with the theme? Ignore?

Reply:
This component belongs to another framework, Ant Design. I think this component will be replaced soon to a correspondent of Material and then will use styles in the same way of another components

Reply:
I'll think about it.

Reply:
We shouldn't replace it, So, in fact, we need to apply the MUI theme.

Reply:
We will use 2 frameworks?